vesc_hw_interface/README.md

```diff
@@ -40,8 +40,17 @@ If your motor unit has other structures, you should implement your own controlle
 - `servo/Kp` (double, *default*: 50.0): proportional gain of the controller.
 - `servo/Ki` (double, *default*: 0.0): integral gain of the controller.
 - `servo/Kd` (double, *default*: 1.0): derivative gain of the controller.
+- `servo/calibration` (bool, *default*: true): if true, the servo will calibrate its origin position.
 - `servo/calibration_current` (double, *default*: 6.0): maximum current used in origin calibration.
 - `servo/calibration_position` (double, *default*: 0.0): the position on which the robot calibrates.
+- `servo/calibration_result_path` (string, *default*: ""): if not empty, the last position will be saved in this path.
+- `servo/last_position` (double, *default*: 0.0): if `calibration` is false, the servo uses this value as its current position.
+
+**Warning**
+
+- If you set `calibration` is true, the servo will move to the limit position with `calibration_current` current. Please make sure that the servo is safe to move.
+- If you set `servo/calibration` is false, the servo will use `servo/last_position` as its current position. If you want to use the last position, you should save the last position in `servo/calibration_result_path` and set `servo/last_position` to the value.
+- If you move the servo manually or by other controllers, you should calibration the servo again.
```

Comment:
If you move the servo manually or use non hardware interface controller or didn't specify the `calibration_result_path`, you should calibrate the servo again.